### ISSUE TITLE
[feature request] swipe from an article to the next one

### ISSUE THREAD

**Issue by symphorien** [bubelov/news#116]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### What feature do you want?

When opening one unread article, I would like to be able to swipe to the right to read the next article in the unread list.

#### Why do you want this feature?

Currently, I need to press back, and then choose one other article. This adds up pretty quickly.

**Comment by gzqx** [comment]
Really want this feature too.

**Comment by nacho-domatix** [comment]
Agree. Just started to use this app and navigation/read is too slow without being able to swipe.

**Comment by peterge1998** [comment]
Please add this, this would be so much more convenient!

**Comment by csolisr** [comment]
I'm currently on the fence about using this app as my main RSS reader because of this issue. There's no way of going to the next article besides of going back and manually clicking on the next item, which adds too much friction. Even a pair of back/forward buttons would go a long way.

**Comment by bubelov** [comment]
Initial support landed in nightly
